srml/executive/src/lib.rs

```diff
@@ -427,6 +428,13 @@ mod tests {
 		type Header = Header;
 		type Event = MetaEvent;
 	}
+	parameter_types! {
+		pub const ExistentialDeposit: u64 = 0;
```

Comment:
You always create 5 types with the same value, in multiple tests, each time just 1 type would be enough.